### DISCUSSION TITLE
如何下载原神

### DISCUSSION THREAD

**Discussion by shydrogen** [share121/inter-knot#2144] [原神]
1、首先搜索进入原神游戏的官网

![ys](https://github.com/user-attachments/assets/b647fa47-46e0-4d79-a717-5e6d6569dc25)

2、进入原神的官网之后在屏幕的中央可以看到ps taptap ios 安卓以及pc客户端选择其中一项进行下载即可

**Comment by tudou927potato** [comment]
我找到你了（斯凯音）

**Reply by shydrogen** [reply]
我这个委托我接了！<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可奸笑" width="75" >

**Comment by tudou927potato** [comment]
![001 (1)](https://github.com/user-attachments/assets/b512a6d7-4188-451d-9dd8-38ce038bb6aa)

**Comment by cleo66** [comment]
<img src="https://bbs-static.miyoushe.com/static/2024/06/12/f2950ab90478f29361959cacadbd123f_3058830200049703740.png" alt="希格雯-打招呼" width="75" ><img src="https://bbs-static.miyoushe.com/static/2024/06/12/f2950ab90478f29361959cacadbd123f_3058830200049703740.png" alt="希格雯-打招呼" width="75" >实用的技巧，感谢楼主

**Comment by yzysrewrwqr** [comment]
别信，要先下载steam，再安装eipc，再导入WeGame，再加装TapTap，就可以进入PS5，用Switch启动，通过steaamdeck里的Xbox组件自动加载，最后我建议用小霸王学习机游玩，体验最佳<img src="https://upload-bbs.miyoushe.com/upload/2022/12/21/6bf4ba03d77560e797085eb691f40b91_4095837989954111024.png" alt="时光交响之章表情包14" width="75" >

**Reply by xingyun-kb** [reply]
6
